src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/client/CommunityApiClient.kt

```diff
@@ -110,6 +111,26 @@ class CommunityApiClient(
     return result
   }
 
+  fun getAllMappaDetails(crn: String): Mono<MappaResponse> {
+    val responseType = object : ParameterizedTypeReference<MappaResponse>() {}
+    return webClient
+      .get()
+      .uri("/secure/offenders/crn/$crn/risk/mappa")
+      .retrieve()
+//      .onStatus(
```

Comment:
Is this supposed to be commented out?

Reply:
Yes :) 
If there are no mappa details available, we should return an empty object and not block the Risk screen with a 404.
That way, other Risk details which are available can still be seen.

Reply:
But why is it commented? Should it not just be deleted or do you plan on adding back in down the line?

Reply:
Done